src/Components/Panels/PanelMenu.tsx

```diff
@@ -166,20 +135,107 @@ export class PanelMenu extends SceneObjectBase<PanelMenuState> implements VizPan
   };
 }
 
+function addVisualizationHeader(items: PanelMenuItem[], sceneRef: PanelMenu) {
+  items.push({
+    text: '',
+    type: 'divider',
+  });
+  items.push({
+    text: 'Visualization',
+    type: 'group',
+  });
+}
+
+function addCollapsableItem(items: PanelMenuItem[], menu: PanelMenu) {
+  const viz = sceneGraph.getAncestor(menu, VizPanel);
+  items.push({
+    text: viz.state.collapsed ? CollapsablePanelType.expanded : CollapsablePanelType.collapsed,
```

Comment:
Seeing the usage here, I'm a bit confused by `CollapsablePanelType`. Is it storing copy? Is it a type? Does it need to be an enum?

Also, maybe more than a collapsable panel type, is a collapsable panel state, that can be expanded or collapsed?

Reply:
Just saw line 155:

```typescript
    const newCollapsable State = viz.state.collapsed ? CollapsablePanel Type .expanded :
```

Reply:
Yeah this is confusing, renamed to CollapsablePanelText: https://github.com/grafana/explore-logs/pull/936/commits/3159461a9996a44be3918afb33d547a49004b807